packages/trader/src/Modules/Trading/Containers/purchase.jsx

```diff
@@ -45,43 +45,30 @@ const Purchase = ({
         const is_disabled = !is_trade_enabled || !info.id || !is_purchase_enabled;
         const is_proposal_error = is_multiplier ? info.has_error && !info.has_error_details : info.has_error;
         const purchase_fieldset = (
-            <div className='trade-params--mobile__payout-container'>
-                {is_vanilla && isMobile() && (
```

Comment:
not needed anymore because we are going to reuse Turbos PayoutPerPointMobile UI element for Vanillas too as discussed with designers.